### PR TITLE
feat(ces): add resource one-click alarm

### PR DESCRIPTION
**Pull request by profoundwu**
**What this PR does / why we need it**:
 add resource one-click alarm
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1.  add resource one-click alarm
```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST="./huaweicloud/services/acceptance/ces" TESTARGS="-run TestAccOneClickAlarm_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ces -v -run TestAccOneClickAlarm_basic -timeout 360m -parallel 4
=== RUN   TestAccOneClickAlarm_basic
=== PAUSE TestAccOneClickAlarm_basic
=== CONT  TestAccOneClickAlarm_basic
--- PASS: TestAccOneClickAlarm_basic (44.56s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ces       44.608s
```

* [x] Documentation updated.
* [x] Schema updated.
* [x] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
    image

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
  image

### CODE REVIEW

File: huaweicloud/services/ces/resource_huaweicloud_ces_one_click_alarm.go [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+	}
+
+	// Successful deletion API call does not guarantee that the resource is successfully deleted.
+	// Call the details API to confirm that the resource has been successfully deleted.
+	_, err = GetOneClickAlarm(deleteOneClickAlarmClient, d.Id())
+	if err != nil {
+		return common.CheckDeletedDiag(d, err, "error retrieving CES one-click alarm")
+	}
+
+	return diag.Errorf("error deleting CES one-click alarm")
```
Comment (saf3dfsa) [inline huaweicloud/services/ces/resource_huaweicloud_ces_one_click_alarm.go:353]: return nil？
```suggestion
	return diag.Errorf("error deleting CES one-click alarm")
```

File: docs/resources/ces_one_click_alarm.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+* `dimension_names` - (Required, List, NonUpdatable) Specifies dimensions in metric and event alarm rules that have
+  one-click monitoring enabled.
+
+  The [dimension_names](#DimensionNames) structure is documented below.
+
+* `notification_enabled` - (Required, Bool) Specifies whether to enable the alarm notification.
+
+* `alarm_notifications` - (Optional, List) Specifies the action to be triggered by an alarm.
+  If the value of `notification_enabled` is **false**, this parameter should not be set.
+  If the value of `notification_enabled` is **true**, this parameter is required.
```
Comment (houpeng80) [inline docs/resources/ces_one_click_alarm.md]:
```suggestion
  + If the value of `notification_enabled` is **false**, this parameter should not be set.
  + If the value of `notification_enabled` is **true**, this parameter is required.
```

File: huaweicloud/services/ces/resource_huaweicloud_ces_one_click_alarm.go [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+	deleteOneClickAlarmPath := deleteOneClickAlarmClient.Endpoint + deleteOneClickAlarmHttpUrl
+	deleteOneClickAlarmPath = strings.ReplaceAll(deleteOneClickAlarmPath, "{project_id}", deleteOneClickAlarmClient.ProjectID)
+
+	deleteOneClickAlarmOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+	deleteOneClickAlarmOpt.JSONBody = buildDeleteOneClickAlarmBodyParams(d)
+	_, err = deleteOneClickAlarmClient.Request("POST", deleteOneClickAlarmPath, &deleteOneClickAlarmOpt)
+	if err != nil {
+		return diag.Errorf("error deleting CES one-click alarm: %s", err)
```
Comment (saf3dfsa) [inline huaweicloud/services/ces/resource_huaweicloud_ces_one_click_alarm.go:343]: add checkdelete for DELETE

File: docs/resources/ces_one_click_alarm.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `description` - The supplementary information about one-click monitoring.
+
+* `enabled` - Whether to enable one-click monitoring.
```
Comment (houpeng80) [inline docs/resources/ces_one_click_alarm.md]:
```suggestion
* `enabled` - Whether the one-click monitoring is enabled.
```

**Pull request merged**